test/reline/helper.rb

```diff
@@ -136,9 +136,14 @@ def input_raw_keys(input, convert = true)
     end
   end
 
-  def assert_line(expected)
-    expected = convert_str(expected)
-    assert_equal(expected, @line_editor.line)
+  def assert_cursor_line(before, after)
```

Comment:
I think `assert_cursor_line` is not clear regarding what it asserts. I think it would be better for this method name to include the meaning of before the cursor and after the cursor.
How about renaming it to `assert_line_around_cursor`?

Reply:
The name `assert_line_around_cursor` sounds good. Thanks, renamed.